tests/test_file.py

```diff
@@ -241,179 +238,12 @@ def test_access_token_credentials(self):
 
         self.assertNotEquals(None, credentials)
         self.assertEquals('foo', credentials.access_token)
-        mode = os.stat(FILENAME).st_mode
+        os.stat(FILENAME).st_mode
```

Comment:
This was an unused variable clean up? It is actually used below on line 248. Previous maintainer probably made a mistake?

To have failures instead of errors, suggest

```python
self.assertTrue(os.path.exists(FILENAME))

if os.name == 'posix':
    mode = os.stat(FILENAME).st_mode
    self.assertEquals('0o600', oct(stat.S_IMODE(mode)))
```

----

PS This is along the lines of "make it better if you must touch it" rather than you being responsible for something you did. Like a living breathing fixit.